### DISCUSSION TITLE
유니티 UI Text에 한글이 입력이 안돼요.

### DISCUSSION THREAD

**Discussion by YNyum** [naverz/zepeto-studio-kor#863] [5. Q&A]
스크립트로 UI Text로 접근해서 한글을 입력하면 깨져서 나오는데 해결방법이 무엇일까요?...

**Comment by YSHFM** [comment]
폰트는 확인해 보셨나요?

**Reply by YNyum** [reply]
기본적으로 제공되는 Arial폰트를 사용 중 입니다.
텍스트에 한글을 직접 입력 후에는 문제 없는데
스크립트로 Text.text  = "안녕하세요" 
이렇게 한글을 넣어주면 런타임에서 깨지는 중입니다.
